actioncable/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "actioncable",
+  "version": "5.0.0-beta4",
+  "description": "WebSocket framework for Rails.",
```

Comment:
We should have this just here, since its not inherent "Rails" means "The Rails" on JS land.